FAQUse: create Buy Anything Button FAQ's (#1655)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

If an input has failed to sign during a previous round it registered to, it will be [temporarily banned](/FAQ/FAQ-UseWasabi.md#why-do-my-coins-occasionally-get-banned-from-participating-in-coinjoin) to prevent denial of service attacks.
Coinjoin coordinators may also reject funds for risk management purposes.
You can view the ban time at the `Wallet Coins` dialog via the search bar or with the keyboard shortcut “CTRL + C + D”.

## Buy Anything Button

### What is the Buy Anything Button?

Since Wasabi version 2.0.5 the Buy Anything Button got added.
This button can be used to buy things with Bitcoin.
The button is an integration with [Shopinbit](https://shopinbit.com/), who are offering their services, now available with an easy click within Wasabi.
It's called the _Buy Anything Button_ because ShopinBit offers many products and a wide variety of services, like a concierge service and travel booking service.

### How does the Buy Anything Button work?

> When clicking on the Buy Anything Button, the user talks with ShopinBit over a new Tor identity (unless Tor is disabled in the Settings).

Reviewer: ```suggestion
When clicking on the Buy Anything Button, the user communicates with ShopinBit over a new Tor identity (unless Tor is disabled in the Settings).
```

The user gets asked some questions, like what he wants to buy, and some follow-up questions needed for the order.
After the details are confirmed in the chat, a bitcoin address gets displayed to pay the invoice.

> Once Shopinbit has received the bitcoin on the address with a confirmation, they will start processing the order.

Reviewer: ```suggestion
Once ShopinBit has received the bitcoin on the address with a confirmation, they will start processing the order.
```

This depends on what has been bought, for example a physical product or a holiday.

### Does the Buy Anything Button hurt my privacy?

> The Buy Anything Button does not compromise Wasabi Wallet wallet's privacy.

Reviewer: ```suggestion
The Buy Anything Button does not compromise your wallet's privacy.
```

Author: applied the suggestions, also made other minor improvements and typo fix

Things like the wallet balance and history are still private.

> However, when using the Buy Anything Button and when having provided some personal details like e-mail and shipping address for the order, then Shopinbit is aware of these, the same as when ordering directly from them or any other merchant.

Reviewer: ```suggestion
However, when using the Buy Anything Button and when having provided some personal details like e-mail and shipping address for the order, then ShopinBit is aware of these, the same as when ordering directly from them or any other merchant.
```

> However, when using the Buy Anything Button and when having provided some personal details like e-mail and shipping address for the order, then ShopinBit is aware of these, the same as when ordering directly from them or any other merchant.

Reviewer: ```suggestion
However, when providing some personal details like e-mail and shipping address for the order, ShopinBit is aware of these, the same as when ordering directly from them or any other merchant.
```

They are aware of this and that's why they ask for the minimal required info possible to complete the order.
You wallet information is never connected to the information used for ordering something.

> But of course, make sure to use private coins when sending them (/anyone) your bitcoin!

Reviewer: ```suggestion
But of course, make sure to use private coins when sending them/anyone your bitcoin!
```

### What can I order and for whom is this available?

ShopinBit can get you anything that is legal in Poland.

> Some services may or may not be available depending on the jurisdiction you're ordering for.

Author: I made this _depending on the jurisdiction_

> Some services may or may not be available depending on the jurisdiction you're ordering for.

Reviewer: ```suggestion
Some services may or may not be available depending on the jurisdiction you're ordering to.
```

### How does the Buy Anything Button work?

When clicking on the Buy Anything Button, the user communicates with ShopinBit over a new Tor identity (unless Tor is disabled in the Settings).
The user gets asked some questions, like what he wants to buy, and some follow-up questions needed for the order.
After the details are confirmed in the chat, a bitcoin address gets displayed to pay the invoice.
Once ShopinBit has received the bitcoin on the address with a confirmation, they will start processing the order.
The processing time depends on what has been bought. 
For example, a physical product may take weeks to deliver but a spontaneous travel booking can be arranged within days.

### Does the Buy Anything Button hurt my privacy?

The Buy Anything Button does not compromise your wallet's privacy.
Things like the wallet balance and history are still private.
However, when providing some personal details like e-mail and shipping address for the order, then ShopinBit is aware of these, the same as when ordering directly from them or any other merchant.
They are aware of this and that's why they ask for the minimal required info possible to complete the order.

> You wallet information is never connected to the information used for ordering something.

Reviewer: @MarnixCroes This PR is waiting for this typo fix so it can be merged.

> You wallet information is never connected to the information used for ordering something.

Author: I forgot about this one
done
